examples/http/aws_billing.js

```diff
@@ -0,0 +1,262 @@
+/**
+ * The driver retrieves billing metrics from Amazon Web Services(AWS) CloudWatch API.
+ * Communication protocol is https.
+ */
+
+//These functions are used to compute hash-based message authentication codes (HMAC) using a specified algorithm.
+function sha256(message) {
+    return D.crypto.hash(message, "sha256", null, "hex");
+}
+
+function hmac(algo, key, message) {
+    key = D._unsafe.buffer.from(key);
+    return D.crypto.hmac(message, key, algo, "hex");
+}
+
+var region = "ADD_REGION";
+var secretKey = "ADD_SECRET_ACCESS_KEY";
```

Comment:
we should probably use D.device. username and password respectively

We do not want the user to store sensitive data in the code